library/src/text/raw.rs

```diff
@@ -197,6 +198,18 @@ impl Finalize for RawElem {
     }
 }
 
+impl LocalName for RawElem {
+    fn local_name(&self, lang: Lang) -> &'static str {
+        match lang {
+            Lang::CHINESE => "代码",
+            Lang::GERMAN => "Computercode",
```

Comment:
Really unsure about the names. The german one is a bit weird. Also, in English, isn't typically `"Listing"` used, at least in LaTeX?

Reply:
I think so too, I took the English one from the template from my uni, but I would agree that there may be more generics words out there. Note however that they are overwritable, and that with I18 that will be fixed once and for all.

Reply:
What would a good german translation be?

Reply:
Difficult. I've search a bit and I didn't find anything better than just also using "Listing" in German.

Reply:
Same with french to be honest, I can't find anything really, weird. I won't try and lookup the other languages for now, it will be fixed with I18n anyway